supabase/functions/generate-resume/index.ts

```diff
@@ -61,12 +61,22 @@ serve(async (req) => {
     ${photo_url ? `- Photo: ${photo_url}` : ""}
     - Personal Particulars:
       - Name: ${user.name}
-      - Age: ${user.age} (include the birth year)
+      - Age: ${user.age}
     - Contact Information:
-      - Phone: 12345678 (phone communication preferred)
-      - Postal code: 460503
-    - Experience:
-      ${user.experience}
+      ${
+        user.phone_number
+          ? `- Phone: ${user.phone_number}  (phone communication preferred)`
+          : ""
+      } 
+      ${user.postal_code ? `- Postal code: ${user.postal_code}` : ""}
+    - Experience: ${user.experience}
+    - Skills:
```

Comment:
I think we should remove the entire skills section for now as we haven't implemented collecting that information.